README.md

```diff
@@ -35,6 +37,7 @@ import (
 )
 
 config := &tfe.Config{
+	Address: "https://tfe.local",
 	Token: "insert-your-token-here",
```

Comment:
I think it may be worth adding `RetryServerErrors: true` to this. In our tests we configure the client this way and it perhaps should be part of best practices when using go-tfe. Although, our default configuration will set `RetryServerErrors` to false, so maybe we can ignore it if we feel it's not necessary.